jupyterlab-dataops/resources/cloudbeaver/drivers/pom.xml

```diff
@@ -0,0 +1,57 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.cloudbeaver</groupId>
+    <artifactId>drivers</artifactId>
+    <version>1.0.0</version>
+    <packaging>pom</packaging>
+
+    <properties>
+        <deps.output.dir>output</deps.output.dir>
+    </properties>
+
+    <modules>
+        <module>bigquery</module>
+        <module>h2</module>
+        <module>mysql</module>
+        <module>mariadb</module>
+        <module>oracle</module>
+        <module>postgresql</module>
+        <module>sqlserver</module>
+        <module>trino</module>
```

Comment:
are all these modules used?

Reply:
That's default config from cloudbeaver.  Only BigQuery is added here. All the others are enabled but we can filter which should be used.